src/screens/MainScreen/styles.js

```diff
@@ -1,19 +1,54 @@
 import { StyleSheet } from 'react-native';
-import { BACKGROUND } from 'constants/colors';
+import { BACKGROUND, BLACK } from 'constants/colors';
+import typography from 'constants/typography';
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: BACKGROUND,
   },
   map: {
-    flex: 5,
+    flex: 7,
```

Comment:
flex 7 is a little wierd

Reply:
For what is this needed?

Reply:
To make the map view's size 7 times the bottom view's (it has a flex 1)